src/zabapgit_persistence.prog.abap

```diff
@@ -1534,6 +1534,11 @@ ENDCLASS.
 CLASS lcl_settings DEFINITION FINAL.
 
   PUBLIC SECTION.
+    CONSTANTS: c_commitmsg_comment_length_dft TYPE i VALUE 50.
+    CONSTANTS: c_commitmsg_comment_length_max TYPE i VALUE 100.
+    CONSTANTS: c_commitmsg_body_size_dft      TYPE i VALUE 72.
+    CONSTANTS: c_commitmsg_body_size_max      TYPE i VALUE 100.
```

Comment:
tried to keep the values near the recommendation, but after some test it seems no one checks or even care about this
I will change this, so it´s up to the user to find suitable max values